db/fetcher/fetcher.go

```diff
@@ -13,27 +13,39 @@ package fetcher
 import (
 	"bytes"
 	"context"
+	"strings"
 
+	"github.com/bits-and-blooms/bitset"
 	dsq "github.com/ipfs/go-datastore/query"
 
 	"github.com/sourcenetwork/defradb/client"
 	"github.com/sourcenetwork/defradb/core"
 	"github.com/sourcenetwork/defradb/datastore"
 	"github.com/sourcenetwork/defradb/datastore/iterable"
 	"github.com/sourcenetwork/defradb/db/base"
+	"github.com/sourcenetwork/defradb/planner/mapper"
+	"github.com/sourcenetwork/defradb/request/graphql/parser"
 )
 
 // Fetcher is the interface for collecting documents from the underlying data store.
 // It handles all the key/value scanning, aggregation, and document encoding.
 type Fetcher interface {
-	Init(col *client.CollectionDescription, fields []*client.FieldDescription, reverse bool, showDeleted bool) error
+	Init(col *client.CollectionDescription, fields []client.FieldDescription,
```

Comment:
todo: Breaking up the props onto multiple lines, but still with multiple props per line is really quite hard to read and I'd much prefer that each prop would have their own line. and not share with the func name and brackets:

```
foo(
    a int,
    b int,
    ...
)
```

Reply:
Yea I didn't like it when I was doing it, but I also hate one line per for interface definitions. But I guess Ill have to, since the both alternatives suck more 🙃